### ISSUE TITLE
Hets Error Output in API

### ISSUE THREAD

**Issue by 0robustus1** [spechub/Hets#1401]
Ok, well in ontohub we'll need to handle errors that are being relayed to us by hets. So we would like to parse the output. However we are using the hets-API, which seems to be problematic as we do not get all necessary information. Here is an example:
- Hets-Call from shell with relative path:  
  `hets -o xml --full-signatures -a none -v2 -O /tmp/ +RTS -K1G -RTS --full-theories syntax-error.clif`
  
  ``` text
  Reading file syntax-error.clif
  Downloaded syntax-error.clif
  **** "syntax-error" (line 2, column 1):
  unexpected end of input
  expecting "(", sentence or ")"
  *** Error: failed to read contents of file: syntax-error.clif
  ```
- Hets-Call from shell with local file-iri:  
  `hets -o xml --full-signatures -a none -v2 -O /tmp/ +RTS -K1G -RTS --full-theories file:///Users/robustus/uni/ontospace/ontohub/syntax-error.clif`
  
  ``` text
  Reading file file:///Users/robustus/uni/ontospace/ontohub/syntax-error.clif
  Downloaded /Users/robustus/uni/ontospace/ontohub/syntax-error.clif
  **** "file:///Users/robustus/uni/ontospace/ontohub/syntax-error" (line 2, column 1):
  unexpected end of input
  expecting "(", sentence or ")"
  *** Error:
  failed to read contents of file: file:///Users/robustus/uni/ontospace/ontohub/syntax-error.clif
  ```
- Hets-Call via api with local file-iri:  
  `curl -XGET http://localhost:8000/dg/file%3A%2F%2F%2FUsers%2Frobustus%2Funi%2Fontospace%2Fontohub%2Fsyntax-error.clif/full-signatures/full-theories`
  
  ``` text
  *** Error:
  failed to read contents of file: file:///Users/robustus/uni/ontospace/ontohub/syntax-error.clif
  ```

So basically when using the Hets-API any information that could be used to determine the cause of error is not being send along to the client. Of course the optimum would be to have structured output (xml or json) which represents the errors, but if we're passing along text it should at least contain the same information one gets when performing a call in the shell/console.
